core/utils/bioinfo_analysis.py

```diff
@@ -69,9 +69,7 @@ def get_bioinfo_analyis_fields_utilization(schema_obj=None):
     if not b_field_objs.exists():
         return b_data
 
-    num_samples_in_sch = core.utils.samples.get_samples_count_per_schema(
-        schema_obj.get_schema_name()
-    )
+    num_samples_in_sch = core.utils.samples.get_samples_count_per_schema(schema_obj.id)
```

Comment:
This would be faster if you simply use schema_obj as i mentioned before:
```suggestion
    num_samples_in_sch = core.utils.samples.get_samples_count_per_schema(schema_obj)
```
```
def get_samples_count_per_schema(schema_obj):
       return core.models.Sample.objects.filter(schema_obj=schema_obj).count()
```